contents/product-engineers/your-first-ab-test.md

```diff
@@ -0,0 +1,86 @@
+
+## When to run your first A/B test?
+
+When you've convinced yourself of the upsides of A/B testing, you must ask **when is the right time** in to start running A/B tests. It can be too early to A/B test. In these cases, it's best to spend your effort elsewhere. There are five rules of thumb to figure if now is the right time for A/B testing:
+
+1. **You have traction.** You aren't early in [your quest for product-market fit](/founders/product-market-fit-game). You're not figuring out what sticks. You have consistent usage and growth and you're looking to optimize it.
+
+2. **Obvious optimizations lead to significant gains.** You improved the areas you want to A/B test on like the home and signup page. The obvious changes made a major improvement to your goal metrics. Obvious optimizations often don't need testing.
+
+3. **What to build next isn't obvious**. You have multiple options for [what to build next](https://newsletter.posthog.com/p/how-we-decide-what-to-build), but data and feedback aren't pointing in a specific direction. You don't have the data needed to make a good decision, and the wrong decision can have significant negative consequences.
+
+4. **Volume is large enough to make a difference.** You have enough users, pageviews, revenue, or whatever your goal is that a 5%-15% percent increase is a big result. For a site with 100s of visitors, a 5% increase in conversion doesn't help much, a 10x increase in visitors does.
+
+5. **Your unit economics support it.** You make money or have a strong idea of how you are going to make money. Increasing the metrics you care about will actually help the business instead of distract from figuring this out.
+
+## What to do before running your first A/B test?
+
+Now that you've decided the time is right, there is a little more setup needed before you dive into running an A/B test. First, if you don't have it already, you need tracking. An A/B test without tracking is pointless. The whole point is to understand the impact changes have, and without tracking you won't know.
+
+Second, you need a system to manage A/B tests. This system needs to store information related to the test, split users into test and control groups, control what variant each group sees in-app, track related behavior, report metrics, and calculate impact.
+
+There are [many tools for doing both](/blog/best-open-source-ab-testing-tools), but we think [PostHog](https://us.posthog.com/signup) is a great option because it combines tracking and A/B testing in [one snippet or SDK installation](/docs/getting-started/install), automatically captures events like pageviews and button clicks, splits users into test and control groups, calculate statistical significance, and more.
```

Comment:
```suggestion
There are numerous [A/B testing tools that do both](/blog/best-open-source-ab-testing-tools), but we think [PostHog](https://us.posthog.com/signup) is a great option because it combines tracking and A/B testing in [one snippet or SDK installation](/docs/getting-started/install), automatically captures events like pageviews and button clicks, splits users into test and control groups, calculate statistical significance, and more.
```